spec/lib/client_summarizer_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe ClientSummarizer do
+  describe "Ncr::WorkOrder" do
+    it "builds summary" do
+      wo = create(:ncr_work_order, amount: 123)
```

Comment:
all the math should now be tested via ClientSummary unit tests.

added Gsa18f::Procurement test in 5a2459d